src/main/java/com/conveyal/gtfs/GTFS.java

```diff
@@ -179,8 +179,8 @@ public static DataSource createDataSource (String url, String username, String p
      * It also lets you run a GraphQL API for all the feeds loaded into the database.
      */
     public static void main (String[] args) throws IOException {
-        // Object mapper used for writing load or validation results to file.
-        ObjectMapper mapper = new ObjectMapper();
+        // Object mapper used for writing load or validation results to file (instantia).
```

Comment:
There appears to be stray text in this comment. This is a detail, but I'd find the code easier to read if the variable were declared and initialized to null just before it is conditionally set to a new instance, around line 211 where we also declare `File directory = null;`